AdventOfCode2021/day2_1/day2_1.py

```diff
@@ -0,0 +1,34 @@
+#!python3
+
+from typing import List
+
+def file_read(file_name) -> List[str]:
+    result: List[str] = []
+    with open(file_name) as f:
+        for line in f:
+            result.append(line)
+    return result
+
+
+def main():
+    input: List[str] = file_read("day2_1.txt")
+    position_data_list: List[str] = []
+    pos_horizontal = 0
+    pos_vertical = 0
+    
+    for i in range(0, len(input)):
+        position_data_list = input[i].split(' ')
+        if ('forward' in position_data_list[0]):
+            pos_horizontal += int(position_data_list[1])
+        elif ('down' in position_data_list[0]):
+            pos_vertical += int(position_data_list[1])
+        elif ('up' in position_data_list[0]):
+            pos_vertical -= int(position_data_list[1])
+        else:
+            print("Something went wrong!")
```

Comment:
If there is more than one line that doesn't match expected format, this line will spam this message. There are two things you can do here. Either (a) add `return` statement after `print`, this way the program will exit if an error is encountered or (b) throw an exception here, this will also cause the program to exit.

You can read more about exceptions online, but that's not necessary.

Reply:
I added `return` for now.

Reply:
I think `return` is needed here.

Reply:
Got it!